scripts/mixins/families/chigoe.lua

```diff
@@ -0,0 +1,26 @@
+-- Chigoe family mixin
+
+require("scripts/globals/mixins")
+
+g_mixins = g_mixins or {}
+g_mixins.families = g_mixins.families or {}
+
+g_mixins.families.chigoe = function(mob)
+    mob:addListener("SPAWN", "CHIGOE_SPAWN", function(mob)
+        mob:hideName(true);
+        mob:untargetable(true);
+        mob:wait(2000);
```

Comment:
remove the wait on the three functions (its only there on sea mobs because they change subanimation and don't move while doing it)